docker/README.md

```diff
@@ -10,47 +10,46 @@ A Docker **image** is a compressed object holding all the dependencies that you
 
 ## Building
 
 To build your own image instead of using the one on Docker Hub, so that it uses your up to date code instead of code on `master`, do
     
-    DOCKER_BUILDKIT=1 docker build --build-arg BUILDKIT_INLINE_CACHE=1 -t avsurfer123/c-runtime:latest -f docker/Dockerfile .
+    docker build -t pierobotics/c-runtime:latest -f docker/Dockerfile .
 
-### Base Image
+If you're on a x64 machine and have the ability to emulate ARM32 either by using Docker Desktop on Windows/Mac or have installed QEMU on Linux (with `docker run --rm --privileged multiarch/qemu-user-static --reset -p yes`), you can instead build an ARM32 image with 
 
-We use a base image to make sure that the building of the actual image is fast. If for some reason the base image needs to be updated and pushed, do
+    DOCKER_CLI_EXPERIMENTAL=enabled docker build --platform linux/arm/v7 -t pierobotics/c-runtime:$TAG -f docker/Dockerfile .
 
-    DOCKER_BUILDKIT=1 cd docker/base && docker build --build-arg BUILDKIT_INLINE_CACHE=1 --cache-from avsurfer123/c-runtime:base -t avsurfer123/c-runtime:base .
+### Base Multi-Architecture Image
 
-If you haven't run this already on your machine and so don't have local caches, it might take up to an hour.
+We use a base image to make sure that the building of the actual image is fast. Our images are also multiarchitecture and building them requires some experimental Docker features to enable `buildx`. If for some reason the base image needs to be updated and pushed, do
+
+    cd docker/base
+    DOCKER_CLI_EXPERIMENTAL=enabled docker buildx create --use   # Only needed the very first time on a computer
+    DOCKER_CLI_EXPERIMENTAL=enabled docker buildx build --platform linux/amd64,linux/arm/v7 --build-arg BUILDKIT_INLINE_CACHE=1 --cache-from pierobotics/c-runtime:base -t pierobotics/c-runtime:base --push .
+
+If you haven't run this already on your machine and so don't have local caches, it will take several hours. Also, this can only be run on x64 machines. If you are using Linux, you will also need to install QEMU before building by doing `docker run --rm --privileged multiarch/qemu-user-static --reset -p yes`.
 
 ## Running
 
-To run the latest Runtime inside a Docker container, do `docker run -it --rm avsurfer123/c-runtime:latest`. Then you can test by in another terminal doing `docker exec -it $(docker ps -q) bash` and running whatever you want inside the container. The `c-runtime` repo will be located at `/root/runtime`.
+To run the latest Runtime inside a Docker container, do `docker run -it --rm pierobotics/c-runtime:latest`. Then you can test by in another terminal doing `docker exec -it $(docker ps -q) bash` and running whatever you want inside the container. The `c-runtime` repo will be located at `/root/runtime`.
 
 ### Devices
 
 If you want to have the container access any Arduino devices, add the flag `--device /dev/ttyACM0` or what ever devices you want to `docker run`.
```

Comment:
I mean no one should be using Windows without WSL lol. On WSL its a different port name but same idea

Reply:
I'm using Windows without WSL and I've been doing fine :  ) I have it installed but I've been using a VM instead, then switched over to Docker and still haven't had a need to use it

What happens if you plug in an Arduino to Windows? Because dev handler searches specifically for ports with files matching the pattern `/dev/ttyACM`

Reply:
On WSL, the devices appear at `/dev/ttyS*`